physionet-django/user/templates/user/credential_application.html

```diff
@@ -75,4 +75,17 @@ <h2>Research Area</h2>
 {% block local_js_bottom %}
 <script src="{% static 'custom/js/enable-tooltip.js' %}"></script>
 <script src="{% static 'user/js/credential-form-control.js' %}"></script>
+<script>
```

Comment:
This change seems unrelated to the original comment in the pull request. It looks like the changes for https://github.com/MIT-LCP/physionet-build/pull/1603 have been unintentionally included here?